ResidentInformationApi.Tests/V1/UseCase/ListContactUseCaseTests.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using ResidentInformationApi.V1.Boundary.Requests;
+using ResidentInformationApi.V1.Boundary.Responses;
+using ResidentInformationApi.V1.Factories;
+using ResidentInformationApi.V1.Gateways;
+using ResidentInformationApi.V1.UseCase;
+using AcademyClaimantInformation = ResidentInformationApi.V1.Domain.AcademyClaimantInformation;
+using HousingResidentInformation = ResidentInformationApi.V1.Domain.HousingResidentInformation;
+using MosaicResidentInformation = ResidentInformationApi.V1.Domain.MosaicResidentInformation;
+
+namespace ResidentInformationApi.Tests.V1.UseCase
+{
+    [TestFixture]
+    public class ListContactUseCaseTests
+    {
+        private Mock<IAcademyInformationGateway> _mockAcademyGateway;
+        private Mock<IHousingInformationGateway> _mockHousingGateway;
+        private Mock<IMosaicInformationGateway> _mockMosaicGateway;
+        private ListContactsUseCase _classUnderTest;
+        private string _academyUrl;
+        private string _housingUrl;
+        private string _mosaicUrl;
+        private readonly Fixture _fixture = new Fixture();
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockAcademyGateway = new Mock<IAcademyInformationGateway>();
+            _mockHousingGateway = new Mock<IHousingInformationGateway>();
+            _mockMosaicGateway = new Mock<IMosaicInformationGateway>();
+            _classUnderTest = new ListContactsUseCase(
+                _mockAcademyGateway.Object,
+                _mockHousingGateway.Object,
+                _mockMosaicGateway.Object
+                );
+            _academyUrl = "http://ACADEMYAPI_URL";
+            _housingUrl = "http://HOUSINGAPI_URL";
+            _mosaicUrl = "http://MOSAICAPI_URL";
+
+            Environment.SetEnvironmentVariable("ACADEMYAPI_URL", _academyUrl);
+            Environment.SetEnvironmentVariable("HOUSINGAPI_URL", _housingUrl);
+            Environment.SetEnvironmentVariable("MOSAICAPI_URL", _mosaicUrl);
+        }
+
+        [Test]
+        public async Task ExecuteReturnsContactInformationResponse()
+        {
+            var stubbedAcademyClaimant = _fixture.Create<AcademyClaimantInformation>();
+            var stubbedHousingResident = _fixture.Create<HousingResidentInformation>();
+            var stubbedMosaicResident = _fixture.Create<MosaicResidentInformation>();
+
+            var academyUrl = _academyUrl;
+            var housingUrl = _housingUrl;
+            var mosaicUrl = _mosaicUrl;
```

Comment:
You don't need these assignments, you can just use `_academyUrl` in place of `academyUrl`

Reply:
You're quite right, I've amended them